src/components/SupportPage/SupportForm.jsx

```diff
@@ -56,10 +56,10 @@ class SupportForm extends React.Component {
     return (
       <React.Fragment>
         {submitFailed && error && this.renderErrorMessage()}
-        {submitSucceeded && this.renderSuccessMessage()}
+        {submitSucceeded && this.renderSuccessMessage}
```

Comment:
Is renderSuccessMessage a boolean? It looks like it's a function since it's bound above

Reply:
Looking at the code I think you'll want to make this a function call again so that https://github.com/edx/frontend-app-admin-portal/blob/063b5298745f3b10bc0e5c179a3a8d46b367920e/src/components/SupportPage/SupportForm.jsx#L33-L44 happens again